python/test/t_QuadraticLeastSquares_std.py

```diff
@@ -1,54 +1,63 @@
 #! /usr/bin/env python
 
 import openturns as ot
+import openturns.testing as ott
 
-
-myFunc = ot.SymbolicFunction(
+f = ot.SymbolicFunction(
     ["x1", "x2"], ["x1*sin(x2)", "cos(x1+x2)", "(x2+1)*exp(x1-2*x2)"]
 )
```

Comment:
This particular function is a poor choice for this unit test because there is no straightforward exact decomposition as a quadratic function. If instead, we choose a quadratic function, when the fact that the off-diagonal coefficients are wrongly estimated cannot get unnoticed. Given that the second test is much better, I suggest to simply remove this part of the unit test since it cannot be maintained anyway.

Reply:
yes, lets do that